rest/route/task.go

```diff
@@ -414,11 +414,11 @@ func (rh *taskSyncPathGetHandler) Run(ctx context.Context) gimlet.Responder {
 	return gimlet.NewTextResponse(t.S3Path(t.BuildVariant, t.DisplayName))
 }
 
-// GET /tasks/{task_id}/set_has_cedar_results
+// POST /tasks/set_has_cedar_results
```

Comment:
I don't think the route signature has to actually change - it can still be POST `/tasks/{task_id}/set_has_cedar_results` and just include any extra info (e.g. the failure/success result) in the payload.

Like [this host route](https://github.com/evergreen-ci/evergreen/blob/47b54b9330bd65824e0a13ad863e0bae4014c0ef/rest/route/host_spawn.go#L31).